### PR TITLE
Improve connectivity-check cue CLI help and documentation

### PR DESCRIPTION
**Pull request by joestringer**
Improve the help text and documentation for listing and generating connectivity checks.

### CODE REVIEW

File: examples/kubernetes/connectivity-check/README.md [hunk 1]
```diff
@@ -37,3 +37,28 @@ multiple files per the following logic:
   directory.
 
 For more information, see https://github.com/cilium/cilium/pull/12599 .
+
+## Listing and generating connectivity checks
```
Comment (sayboras) [inline examples/kubernetes/connectivity-check/README.md:41]: not related to this PR, but I just saw one typo in line 36 (e.g. definitons -> definitions)

File: examples/kubernetes/connectivity-check/README.md [hunk 2]
```diff
@@ -37,3 +37,28 @@ multiple files per the following logic:
...
+
+```
+$ make help
+List connectivity-check resources specified in this directory
+
+Usage:
+  cue [-t component=<component>] [-t kind=<kind>] [-t name=<name>] [-t quarantine=true] [-t topology=<topology>] [-t traffic=any] [-t type=<tooltype>] <command>
```
Comment (sayboras) [inline examples/kubernetes/connectivity-check/README.md:48]: this is really helpful 👍 .

Just a quick question on this. `make help` output suggests to run `cue` command directly, meanwhile our Makefile is running cue in docker. 

So do you feel that we need to add one small step to install cue binary in local system (e.g. similar to what we have for golangci-lint, ginkgo, gomega, etc) ?

**Review by sayboras — APPROVED** [review]
LGTM 💯 

I nominated this PR for 1.8 backported as well.

**Comment by joestringer** [comment]
@sayboras addressed feedback, thanks for the review!

**Review by christarazi — APPROVED** [review]

**Pull request merged**